hbw/app/controllers/hbw/api_controller.rb

```diff
@@ -0,0 +1,7 @@
+module HBW
+  class ApiController < BaseController
+    rescue_from HBW::Remote::RemoteError do |exception|
+      render json: exception.to_s.force_encoding('utf-8').to_json, status: 504
```

Comment:
Any `RemoteError` is a `Gateway Timeout`? Why do you so sure?

Reply:
All actions causing `RemoteError` are of the same type and tell us about the unavailability of the camunda service. Backend HBW proxies the error to the front with detailed description.